### ISSUE TITLE
Costa Rica - Incorrect Data

### ISSUE THREAD

**Issue by Guilherme-B** [whosonfirst-data/whosonfirst-data#1856]
Costa Rica's data seems to be very outdated. For instance, the county Renacimiento doesn't seem to exist anywhere, not sure where this ever game from. Same for Bugaba, among many others.
- 421181903 (Renacimiento)
- 421170137 (Bugaba)
- 421188117 (Esparza)

There's even a County named Buenos Aires (421201609).

[Provinces](http://daticos-geotec.opendata.arcgis.com/datasets/6e2ec42966d54e7087d9da25ad78abe0_0)
[Districts](http://daticos-geotec.opendata.arcgis.com/datasets/741bdd9fa2ca4d8fbf1c7fe945f8c916_0)
[Counties](http://daticos-geotec.opendata.arcgis.com/datasets/249bc8711c33493a90b292b55ed3abad_0?geometry=-92.146%2C8.478%2C-75.458%2C11.723)
[Poblados](http://daticos-geotec.opendata.arcgis.com/datasets/f3c43f4bcf884657aea5683e2041ebea_0?geometry=-87.439%2C9.227%2C-79.095%2C10.85)

The data is licensed for Public Use.

Any chance this can be updated?

**Comment by Guilherme-B** [comment]
As far as I can tell, the errors seem to be in the 3rd administrative level, namely, the following entities don't seem to exist (at least, by their current name):

421201609
421202609
890442901
421190543
421194119
421175399
421177415
421180917
421181903
421188113
421188117
421170137

**Comment by stepps00** [comment]
Nice find, @Guilherme-B - thanks for the links.

Agreed that we should update the admin data in Costa Rica, there are likely updated to make and outdated records we should update.

The links you provided seem to contain data for placetypes we'd need in order to update records, my only question is around the data licenses. The licenses on the arcgis page suggest the datasets are "Public Use", but we'd need to find more specifics about the license type and terms. I briefly poked around the [source website](https://www.geotecnologias.com/es-cr/home), which led me to:

- https://geodatos.maps.arcgis.com/home/index.html
- http://daticos-geotec.opendata.arcgis.com/

Do you happen to know if there are more specific license terms for each dataset? I'll keep looking, too.

**Comment by Guilherme-B** [comment]
Thanks for the reply as usual @stepps00 .

As for the license, I'm not sure to be honest, I can e-mail them in Spanish and keep trying to lookup for the answer either way. What type of license does WOF require?

**Comment by Guilherme-B** [comment]
Here's an alternative from the official government source, [WMS](http://geos.snitcr.go.cr/be/IGN_5/wms?), [WFS](http://geos.snitcr.go.cr/be/IGN_5/wfs?). By looking at the [capabilities ](http://geos.snitcr.go.cr/be/IGN_5/wms?request=GetCapabilities) request, the data to be retrieved is:
* "limiteprovincial_5k" for provinces
* "limitedistrital_5k" for districts
* "limitecantonal_5k" for cantons

Here is the [metadata](http://www.snitcr.go.cr/Metadatos/full_metadata?k=Y2FwYTo6SUdOXzU6OmxpbWl0ZWNhbnRvbmFsXzVr) for the layers. The copyright is set to "copyright", which corresponds to:

"1) Derechos de autor (copyright):
Es el derecho exclusivo de publicación, producción o venta de los derechos de un trabajo literario, dramático, musical o artístico, o de uso de una marca comercial o etiqueta, garantizado por ley para un período específico de tiempo a un autor, compositor, artista o distribuidor."
or, via google translate:
"1) Copyright (copyright):
It is the exclusive right of publication, production or sale of the rights to a literary, dramatic, musical or artistic work, or the use of a trademark or label, guaranteed by law for a specific period of time to an author, composer, artist or distributor"
Does this prevent WOF usage?

**Comment by stepps00** [comment]
> What type of license does WOF require?

WOF requires a `CC-BY` or `CC0` equivalent license for import.

I'm not sure WOF could use that license to import.. it's not clear to me if there is a specific license to each of these datasets (that looks like a general "copyright"). I will also poke around and paste any admin data for Costa Rica in this issue.

**Comment by justinelliotmeyers** [comment]
could always email their statistical bureau - super nice people.

**Comment by nvkelso** [comment]
That sounds like boilerplate wrapper text to me, let's definitely contact them :)

**Comment by Guilherme-B** [comment]
> could always email their statistical bureau - super nice people. https://www.inec.cr/

I did look for the data there, had no luck retrieving it.

Well, I assume you will e-mail them on behalf of WOF, let us know if they reply :)

**Comment by stepps00** [comment]
I reached out; I'll post any info they forward.

**Comment by justinelliotmeyers** [comment]
I also emailed (*en espanol)!  Hopefully we hear back soon

**Comment by justinelliotmeyers** [comment]
emailed agency responsible for license preguntas: SecretariaIGN@rnp.go.cr

**Comment by justinelliotmeyers** [comment]
[CONNECTING SERVICES IN QGIS AND ARCGIS.pdf](https://github.com/whosonfirst-data/whosonfirst-data/files/4827130/CONNECTING.SERVICES.IN.QGIS.AND.ARCGIS.pdf)

replied asking for terms

**Comment by Guilherme-B** [comment]
Just found duplicate entities:
- 421203517
- 1125389441
- 421173725
- 421185049
- 421185053
- 421166603

Could be a good idea to add to the ingestion/validation pipeline a check to see if the same parent contains direct children with the same name - as its unlikely to happen, just so it can be manually checked.

**Comment by Guilherme-B** [comment]
> [CONNECTING SERVICES IN QGIS AND ARCGIS.pdf](https://github.com/whosonfirst-data/whosonfirst-data/files/4827130/CONNECTING.SERVICES.IN.QGIS.AND.ARCGIS.pdf)
> 
> replied asking for terms

Thanks for handling this :)

**Comment by justinelliotmeyers** [comment]
![Screenshot_20200624-145055](https://user-images.githubusercontent.com/5324316/85621016-3dcdca80-b62a-11ea-8321-d6cef3c4319b.png)

**Comment by stepps00** [comment]
Correct. If we're going to import geometries or properties from a source into WOF, that source license must allow commercial and non-commercial use. It looks like the SNIT source prohibits commercial use, unfortunately.

**Comment by Guilherme-B** [comment]
> Correct. If we're going to import geometries or properties from a source into WOF, that source license must allow commercial and non-commercial use. It looks like the SNIT source prohibits commercial use, unfortunately.

Its never easy is it? The original [link I shared](http://daticos-geotec.opendata.arcgis.com/datasets/6e2ec42966d54e7087d9da25ad78abe0_0) doesn't specify a usage permission, and I think by default ESRI shared data is allowed for noncommercial usage only. We can always [email the group that shares the data](https://www.geotecnologias.com/es-cr/home) on ESRI asking them if they would consider specifying a license in the ESRI portal - which then supersedes ESRI's.

**Comment by Guilherme-B** [comment]
Starting to think the only way for data to be used is for someone to make the maps by hand and share them as their own work hah.

**Comment by Guilherme-B** [comment]
Starting to think the only way for data to be used is for someone to make the maps by hand and share them as their own work hah.

**Comment by Guilherme-B** [comment]
@justinelliotmeyers  I did notice you have this data on your [repo](https://github.com/justinelliotmeyers/official_costa_rica_administrative_boundary_shapefile), what is the source?

**Comment by justinelliotmeyers** [comment]
@Guilherme-B SNIT

@stepps00 potential new data! https://revolutionarygis.wordpress.com/2020/07/16/localidades-de-costa-rica/  I did not check license.

**Comment by Guilherme-B** [comment]
> @stepps00 potential new data! https://revolutionarygis.wordpress.com/2020/07/16/localidades-de-costa-rica/ I did not check license.

Great news! I had stumbled upon that blog as well, especially since there is data for plenty of countries. Did you manage to find a contact button? Can't seem to find one available.

**Comment by Guilherme-B** [comment]
Just an update: the datasets shared in [ArcGis](http://daticos-geotec.opendata.arcgis.com/datasets/741bdd9fa2ca4d8fbf1c7fe945f8c916_0) are distributed under a Public license, which, according to [creativecommons](https://creativecommons.org/share-your-work/public-domain/pdm/) gives you free right of usage.
